src/com/softwaret/kpdf/repository/validation/input/login/LoginValidationRepositoryImpl.kt

```diff
@@ -0,0 +1,11 @@
+package com.softwaret.kpdf.repository.validation.input.login
+
+class LoginValidationRepositoryImpl : LoginValidationRepository {
+
+    override fun validateLogin(login: String?) =
```

Comment:
Teraz tak jeszcze myślę, czy nie jesteśmy zbyt para-Kotlinowi zakładając, że jeżeli jest error to jest `null`. Sądzę, że moglibyśmy tutaj wykorzystać Kotlinowe funkcjonalności takie jak `require`, czy `requireNotNull` i potem łapać wyjątki, które będą mapowane na odpowiedni błąd. Zaproponuję do tego rozwiazanie i wystawię na Ciebie, abyś zerknął czy to ma większy sens. Chodzi mi po prostu o nie rzucanie nullami na lewo i prawo gdy tylko coś nie idzie tak jak powinno.

Reply:
Możemy to jutro obgadać, ale proponuję zrobić tak: 

- Exceptiony dla niespodziewanych zachowań które zwrócą 500 (dzielenie przez zero, coś się mocno zesrało w configu albo doszło do sytuacji której nie obsługujemy w when) 
- Jakieś Try albo Either dla spodziewanych błędów